### PR TITLE
Reverse roll fuse

### PR DESCRIPTION
**Pull request by wwbitejotunn**
### PR types
Performance optimization

### PR changes
OPs

### Describe
fuse reverse circlic shift in window attention in swin model.
it speed up swin tiny model from 0.92ms to 0.89ms in A10

### CODE REVIEW

File: paddle/fluid/inference/tensorrt/op_teller.cc [hunk 1]
```diff
@@ -2347,6 +2347,7 @@ struct SimpleOpTypeSetTeller : public Teller {
       "unsqueeze2",
       "fused_token_prune",
       "layernorm_shift_partition",
+      "reverse_roll",
```
Comment (b3602sss) [inline paddle/fluid/inference/tensorrt/op_teller.cc:2571]: 只有dynamic的时候才可以

Comment (b3602sss) [reply paddle/fluid/inference/tensorrt/op_teller.cc:2571]: 一般int8 set 也会加上

Comment (wwbitejotunn) [reply paddle/fluid/inference/tensorrt/op_teller.cc:2571]: Done, thanks~

**Review by b3602sss — DISMISSED** [review]

File: paddle/fluid/inference/tensorrt/op_teller.cc [hunk 2]
```diff
@@ -2526,10 +2537,11 @@ struct SimpleOpTypeSetTeller : public Teller {
       "squeeze2",
       "unsqueeze2",
       "fused_token_prune",
+      "preln_layernorm_shift_partition",
       "layernorm_shift_partition",
+      "reverse_roll",
       "tanh_shrink",
       "logsigmoid",
-      "preln_layernorm_shift_partition",
```
Comment (b3602sss) [inline paddle/fluid/inference/tensorrt/op_teller.cc]: 不要删

Comment (wwbitejotunn) [reply paddle/fluid/inference/tensorrt/op_teller.cc]: 次序换回来了

**Review by b3602sss — DISMISSED** [review]

**Review by b3602sss — DISMISSED** [review]

**Review by b3602sss — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by XieYunshen — APPROVED** [review]
LGTM for 
`set_tests_properties(test_reverse_roll_fuse_pass PROPERTIES TIMEOUT 120)`

**Pull request merged**